app/queries/consumer_apps/rpush_app_query.rb

```diff
@@ -16,24 +17,24 @@ def initialize(app_bundle:, platform:)
 
     def call
       if consumer_app.android?
-        android_app = Rpush::Gcm::App.where(name: app_bundle).first
+        android_app = Rpush::Gcm::App.where(name: app_name).first
         android_app || recreate_android_app!
       elsif consumer_app.ios?
-        ios_app = Rpush::Apns2::App.where(name: app_bundle).first
+        ios_app = Rpush::Apns2::App.where(name: app_name).first
```

Comment:
Queries now use a name that combines `app_bundle` and `platform`, which was causing the conflict when looking up the apps.